src/AccessibilityInsights/MainWindow.xaml

```diff
@@ -234,7 +234,7 @@
                             </ItemsControl.ItemsPanel>
                             <Button Width="35" Height="28"
                                     Style="{StaticResource BtnStandard}"
-                                    x:Name="btnHelp" Focusable="False" IsTabStop="False"
```

Comment:
This change is a lot easier if you hide whitespace changes (accessible via the gear icon):

![image](https://user-images.githubusercontent.com/45672944/76913079-305ab480-6873-11ea-95d3-873494b3fe2f.png)